### DISCUSSION TITLE
Storybook migration to lastest

### DISCUSSION THREAD

**Discussion by vsirenko** [storybookjs/storybook#24870] [Help]
Before migration i had a this decorator:
`export const decorators = [
  (Story) => (
    <StorybookWrapper>
      <Story />
    </StorybookWrapper>
  ),
];`

<img width="742" alt="image" src="https://github.com/storybookjs/storybook/assets/81309430/fa78eb60-47c3-404b-a08f-9eaaadf5d9c1">
    
    and now i have infinity loading:
    
    
![image](https://github.com/storybookjs/storybook/assets/81309430/b11d5c38-26cd-43b5-a38f-9ccb3029194a)

**Comment by valentinpalkovic** [comment]
Can you try to change:

```tsx
export const decorators = [ (Story) => ( <StorybookWrapper> <Story /> </StorybookWrapper> ), ];
```

to:

```tsx
export const decorators = [ (Story) => ( <StorybookWrapper>{Story()}</StorybookWrapper> )];
```

**Reply by vsirenko** [reply]
same error, after reload have infinite  loader

**Reply by valentinpalkovic** [reply]
Can you provide a reproduction?
